### ISSUE TITLE
Question: Nuitka compression use Zstandard, is there any options to use some other compression tools beside Zstandard?

### ISSUE THREAD

**Issue by andizaa** [Nuitka/Nuitka#2409]
Nuitka compression use Zstandard, is there any options to use some other compression tools beside Zstandard?
I have an issue, I tried compiling my python code using Nuitka, when I install zstandard which is the compression method, the exe file got quarantined by Avast Antivirus. It's so weird. I tried to uninstall zstandard, and the exe doesn't get quarantined by Avast, but without compression the file size is too big. I dont know why zstandard makes the exe quarantined by Avast, but is there a way to put some other alternatives for compression besides zstandard?
thanks .

**Comment by kayhayen** [comment]
You can disable the compression only. The AV situation is really bad. You need to follow instructions in Nuitka commercial to mitiigate it somewhat. But indeed, the standalone form is barely flagged in comparison. I wonder if adding a offline decompressor will help. The file format is going to undergo changes in 1.9 release that hopefully make it an actual archive format with per file compression that will also enhance performance.

**Comment by kayhayen** [comment]
I implemented per file compression with `--onefile-archive` but I don't think I will keep it. The main goal is not compile time savings, but start time savings, and what I realized, is that with yet another archive format change, where the meta information is all at start of file, and then caching can detect that the later parts are all unneeded, should make it such, that for mere launching, most of the compressed data is never uncompressed, this point becomes moot.

So, while this is now implemented, I guess it's not going to say. Still closing it as the desire of yours seems more or less addressed, and for quicker turnarounds disabling compression is the way to go I guess.
